config/main.py

```diff
@@ -313,6 +313,14 @@ def _abort_if_false(ctx, param, value):
     if not value:
         ctx.abort()
 
+def _get_optional_services():
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    optional_services_dict = config_db.get_table('FEATURE')
+    if not optional_services_dict:
+        return None
+    return optional_services_dict.keys()
```

Comment:
Maybe this could return the entire dictionary, then the consumers below can determine whether or not to stop/start the service based on whether it is enabled or disabled in the DB, in conjunction with the result of running `systemctl is-enabled`. Theoretically, the two states should match, but I'm trying to consider the corner case just in case they are out-of-sync.

Thoughts?

Reply:
If out of sync I prefer the result from `systemctl is-enable`. In general this out of sync should be caused by user behavior such as directly running `systemctl enable xxx`.  This is the reason I choose `systemctl is-enabled` instead of config-db.
Anyway I am open for this. Giving priority to config-db is acceptable.

Reply:
@jleveque any comment? If no could you approve this PR?

Reply:
I am fine with this approach. I will approve the PR, but I would also like to get @lguohan's approval before merging.

Reply:
Thanks